sqswatcher/plugins/sge.py

```diff
@@ -19,28 +17,61 @@
 import socket
 import logging
 import shlex
+from sqswatcher.sqswatcher import RemoveHostFromSchedulerException
```

Comment:
We could think to move these exceptions in an utility module, imported by both `sqswatcher.py` and `sge.py` modules.